core/code/dialog_about.js

```diff
@@ -0,0 +1,179 @@
+    + '<div>'
+    + '  <a href="'+'@url_homepage@'+'" target="_blank">IITC Homepage</a> |' +
+    '  <a href="'+'@url_tg@'+'" target="_blank">Telegram channel</a><br />'
+    + '   On the script’s homepage you can:'
+    + '   <ul>'
+    + '     <li>Find Updates</li>'
+    + '     <li>Get Plugins</li>'
+    + '     <li>Report Bugs</li>'
+    + '     <li>Contribute!</li>'
+    + '   </ul>'
+    + '</div>'
+    + '<hr>'
+    + '<div>Version: ' + getIITCVersion() + '</div>';
+
+  if (isShortOnLocalStorage()) {
+    html += '<div class="warning">You are running low on LocalStorage memory.<br/>Please free some space to prevent data loss.</div>';
+  }
+
+  if (window.isApp && app.getVersionName) {
+    html += '<div>IITC Mobile ' + app.getVersionName() + '</div>';
+  }
+
+  var plugins = getPlugins();
+  if (plugins) {
+    html += '<div><p>Plugins:</p><ul>' + plugins + '</ul></div>';
+  }
+
+  return html;
+}
+
+
+function getPlugins() {
+  var pluginsInfo = window.bootPlugins.info;
+
+  while (window.bootPlugins[0]) {
+    var plugin = window.bootPlugins.shift();
+    pluginsInfo.push(plugin.info || {});
+  }
+
+  var extra = getIITCVersionAddition();
+
+  var plugins = pluginsInfo.map(convertPluginInfo)
+    .sort(function (a, b) { return a.name > b.name ? 1 : -1; })
+    .map(function (p) { return pluginInfoToString(p, extra); })
+    .join('\n');
+
+  return plugins;
+}
+
+function convertPluginInfo(info, index) {
+  // Plugins metadata come from 2 sources:
+  // - buildName, pluginId, dateTimeVersion: inserted in plugin body by build script
+  //   (only standard plugins)
+  // - script.name/version/description: from GM_info object, passed to wrapper
+  //   `script` may be not available if userscript manager does not provede GM_info
+  //   (atm: IITC-Mobile for iOS)
+  var result = {
+    build: info.buildName,
+    name: info.pluginId,
+    date: info.dateTimeVersion,
+    error: info.error,
+    version: undefined,
+    description: undefined
+  };
+
+  var script = info.script;
+  if (script) {
+    if (typeof script.name === 'string') {
+      result.name = script.name.replace(/^IITC[\s-]+plugin:\s+/i, ''); // cut non-informative name part
+    }
+    result.version = script.version;
+    result.description = script.description;
+  }
+
+  if (!result.name) {
+    if (script_info.script) { // check if GM_info is available
+      result.name = '[unknown plugin: index ' + index + ']';
+      result.description = "this plugin does not have proper wrapper; report to it's author";
+    } else { // userscript manager fault
+      result.name = '[3rd-party plugin: index ' + index + ']';
+    }
+  }
+
+  return result;
+}
+
+function pluginInfoToString(p, extra) {
+  var info = {
+    style: '',
+    description: p.description || '',
+    name: p.name,
+    verinfo: formatVerInfo(p, extra)
+  };
+
+  if (isStandardPlugin(p)) {
+    info.style += 'color:darkgray;';
+  }
+
+  if (p.error) {
+    info.style += 'text-decoration:line-through;';
```

Comment:
maybe css classes to avoid putting style into the code ?

Reply:
done